### ISSUE TITLE
4 streams: Only one video is playing

### ISSUE THREAD

**Issue by Nuubus** [NoahFetz/F1AppleTV#27]
Which version of OS are you running this on? I'm having issues where the other streams would freeze.

I've tried pausing for a bit and fast forwarding/rewind and it seems to start to control the next stream over.

Running OS 14.6 Beta 2 =X

**Comment by NoahFetz** [comment]
Are you still having this issue?

**Comment by Nuubus** [comment]
Yeah it still continues to happen. I think I figured out what possibly could be happening. 

- I started to rearranging streams. It may still think it's in its original position.

- i.e. 1 - Norris, 2 - Riccardo, 3 - Data Feed, 4 - Main feed
You stop/start stream 1. It will start playing audio feed from 4-Main Feed


I know this may be a odd bug but not a big deal. I closed the app and restarted it. As long as I set it up from the beginning it works fine. Think it has to do with when you're mid stream and re-arrange.
